mohitspawar-frisco-tx.md

```diff
@@ -0,0 +1,6 @@
+# Frisco TX
+
+## Place to eat
+- Nerdvana
+- Hutchin's BBQ
+- The Red Lantern
```

Comment:
This sounds like a place that has great 🍔 s, if not, where should I go to get one?